managed/src/main/java/com/yugabyte/yw/models/Region.java

```diff
@@ -88,50 +88,50 @@ public void setActiveFlag(Boolean active) {
 
   @DbJson
   @Column(columnDefinition = "TEXT")
-  public JsonNode details;
+  public Map<String, String> details;
```

Comment:
Create a POJO  `RegionDetails` with two properties `sg_id` and `vnet`. (Just public static inner class is fine)  And use that class here.
Fix the below getters and setters to work with that POJO.

Reply:
This is low priority btw becaue I am hiding this property and only exposing the setters and getters below. But It is a simple fix so you may want to do it in this PR.

Reply:
if it only can contains above two fields, i will make the change. My local has only one field for the region i have. So made it map.